package.json

```diff
@@ -59,12 +59,15 @@
     "babel-core": "7.0.0-bridge.0"
   },
   "dependencies": {
+    "@formatjs/intl-pluralrules": "1.3.4",
+    "@formatjs/intl-relativetimeformat": "4.4.3",
     "chalk": "2.4.2",
     "compression": "1.7.4",
     "connected-react-router": "6.5.2",
     "cross-env": "6.0.3",
     "express": "4.17.1",
     "fontfaceobserver": "2.1.0",
+    "full-icu": "1.3.0",
```

Comment:
Should this be a devDependency?

Reply:
The install instructions on https://www.npmjs.com/package/full-icu are displaying the commands for putting it in dependencies (no -D flag), but I am open to move it to the devDependencies, if desired.

I hope we can expect some activity from @julienben soon?

Reply:
Hope to check soon. I have a problem with upgrade intl to v3